src/FileConventions/Library.fs

```diff
@@ -0,0 +1,25 @@
+﻿module FileConventions
+
+open System.Linq
+open System.IO
+open System
+open Mono.Posix

```

Comment:
@realmarv please separate namespaces in groups like our CONTRIBUTING.md guidelines state

Reply:
Done